sklearn/impute.py

```diff
@@ -452,24 +452,34 @@ class ChainedImputer(BaseEstimator, TransformerMixin):
         "random"
             A random order for each round.
 
-    n_imputations : int, optional (default=100)
-        Number of chained imputation rounds to perform, the results of which
-        will be used in the final average.
+    n_iter : int, optional (default=10)
+        Number of imputation rounds to perform before returning the final
+        imputations computed during the final round. A round is a single
+        imputation of each feature with missing values.
 
-    n_burn_in : int, optional (default=10)
-        Number of initial imputation rounds to perform the results of which
-        will not be returned.
-
-    predictor : estimator object, default=BayesianRidge()
+    predictor : estimator object, default=BayesianRidge() or RidgeCV()
         The predictor to use at each step of the round-robin imputation.
-        It must support ``return_std`` in its ``predict`` method.
+        If ``sample_posterior`` is True, the predictor must support
+        ``return_std`` in its ``predict`` method. Also, if
+        ``sample_posterior=True`` the default predictor will be
+        ``BayesianRidge()`` and ``RidgeCV`` otherwise.
+
+    sample_posterior : boolean, default=False
+        Whether to sample from the (Gaussian) predictive posterior of the
+        fitted predictor for each imputation. Predictor must support
+        ``return_std`` in its ``predict`` method if set to ``True``. Set to
+        ``True`` if using ``IterativeImputer`` to have the same functionality
+        as MICE (Multivariate Imputation by Chained Equations).
 
     n_nearest_features : int, optional (default=None)
         Number of other features to use to estimate the missing values of
```

Comment:
just a typo here: 'of each' instead of 'of the each'

Reply:
Thanks!